crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -80,6 +80,12 @@ def cancel
     render json: { error: e.message }, status: :unprocessable_entity
   end
 
+  api :GET, "/api/upgrade/repocheck", "Check for missing node repositories"
```

Comment:
we could also adapt the response of `/api/crowbar/repocheck` to look the same. that would be better for using it in `crowbarctl` 

btw https://github.com/crowbar/crowbar-client/pull/104 adapts `crowbarctl` to the new node repocheck